### DISCUSSION TITLE
Low transcriptome cells

### DISCUSSION THREAD

**Discussion by millerkaplanlab** [GreenleafLab/ArchR#1826] [General]
With our scRNA-seq analysis, we are able to identify those cells with low transcripts and cluster them out. Does ArchR provide a similar feature whereby low transcript cells can be identified?

**Comment by rcorces** [comment]
this information is stored in `cellColData`

**Comment by millerkaplanlab** [comment]
Thank you. Are there any parameters that can be changed to remove low transciptome cells when importing the 10x feature matrix.

**Reply by rcorces** [reply]
no. remove afterwards with `subsetArchRProject()`
